app/controllers/application_controller.rb

```diff
@@ -57,12 +58,32 @@ def with_current_request
     end
   end
 
+  # Validates against openapi.json
+  # - only for HTTP POST/PATCH
```

Comment:
@slemrmartin the validation validates also parameters, etc. so it should be used for GET request too